.changeset/calm-geese-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes': patch
+---
+
+refactor kubernetes error detection to make way for proposed solutions
```

Comment:
Based on the API report this looks like a breaking change, so I'd expect this to be `minor` along with calling out the breaking changes.

Reply:
Will fix

Reply:
done